buildconfig/pygame-stubs/math.pyi

```diff
@@ -0,0 +1,84 @@
+from typing import Optional, Union, Tuple, List, overload
+class VectorElementwiseProxy: ...
+
+def enable_swizzling() -> None: ...
+def disable_swizzling() -> None: ...
+
+class Vector2:
+    x: float
+    y: float
+    def __init__(self, x: Optional[Union[float, Vector2, Tuple[float, float], List[float]]]=0, y: Optional[float]=0) -> None: ...
+    def dot(self, other: Vector2) -> float: ...
+    def cross(self, other: Vector2) -> Vector2: ...
+    def magnitude(self) -> float: ...
+    def magnitude_squared(self) -> float: ...
+    def length(self) -> float: ...
+    def length_squared(self) -> float: ...
+    def normalize(self) -> Vector2: ...
+    def normalize_ip(self) -> None: ...
+    def is_normalized(self) -> bool: ...
+    def scale_to_length(self, value: float) -> None: ...
+    def reflect(self, other: Vector2) -> Vector2: ...
+    def reflect_ip(self, other: Vector2) -> None: ...
+    def distance_to(self, other: Vector2) -> float: ...
```

Comment:
other can be any sequence, not just Vector2. See vector.c@_vector_distance_helper

Reply:
Tnx